### PR TITLE
fix-login-in-about

### PR DESCRIPTION
**Pull request by urirot**

### CODE REVIEW

**Comment by c-simpson** [comment]
One concern I have is that I don't get the login screen if time expires.  
To test this, hit the URL
`http://localhost/api/user/timeout/45` twice (to get both instances) before logging in [1].
Then log-in and ignore the popup - let the timer run out -> there's no login form.

[1] If you are already logged-in, go to browser Dev Tools and run `sessionStorage.clear()` then reload the page.

**Comment by urirot** [comment]
@c-simpson Fixed. let me know if it's working well on your end now. 
in my case, i ran `localhost:3000` instead of `localhost`.

**Comment by c-simpson** [comment]
I think the issue is that in App.js , the About route is not in the Switch, so you can get both  `<About \>`  and  `<Login \>`  rendered on the page.

**Comment by urirot** [comment]
good catch. all should be working now

File: src/client/src/App.js [hunk 1]
```diff
@@ -88,25 +88,29 @@ function AuthenticatedApp() {
     const hdr = userRole === 'admin' ? <AdminHeader/> : <Header/> // If we're going to display a header, which one?
 
     const history = useHistory();
+    debugger;
```
Comment (c-simpson) [inline src/client/src/App.js:91]: Wonder if we could do a post-commit hook to catch these?

Comment (urirot) [reply src/client/src/App.js:91]: oh probably, but it doesn't have any effect if dev tool isn't open, so don't think we need it

**Review by c-simpson — APPROVED** [review]
Looks good.

**Pull request merged**